cloudstate/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+Copyright 2020 Lightbend Inc.
+Licensed under the Apache License, Version 2.0.
+"""
+
+from .version import __version__
```

Comment:
note: this is needed so that setup.py detects this directory as a module directory.